### PR TITLE
Feature #17 eclass application classification classes

### PR DESCRIPTION
**Pull request by BirgitBoss**
How to reference an ECLASS application or classification class? #17

### CODE REVIEW

**Comment by aorzelskiGH** [comment]
Check again, after talking with ECLASS experts

**Comment by aorzelskiGH** [comment]
Check wether the asset category of the Technical Data submodel template can be used.

File: README.md [hunk 1]
```diff
@@ -162,8 +162,42 @@ Via display names (introduced in V3.0RC01 Referable/displayName, it is also poss
...
 Note: It is requested that the idShort of a non-identifiable is unique in its name space (for example in a Submodel or in a SubmodelElementCollection). This is not requested for the display names!
 
-
-
+**[How to reference an ECLASS application or classification class](#id45)** <a id="id45"></a>
+
+Referencing of application and classification classes is no different to referencing the ECLASS property definition.
+Since classification and application classes correspond to a set of properties they can be used to describe the semantics of a complete submodel or of an SubmodelElementCollection.
+
+Please be aware: to be completeley unambiguous it is either necessary to model the complete hierarchy in the submodel or by using a semanticId with more than one IRDI. The keys in the reference are ordered.
```
Comment (BirgitBoss) [inline README.md:170]: In ECLASS a classification class can only be contained in exactly another classification class. IEC61360 would also allow that a classification class is member of more than one other classification class. So for ECLASS property together with its parent classification class is sufficient for a unique semanticId. ==> change example

File: README.md [hunk 2]
```diff
@@ -162,8 +162,42 @@ Via display names (introduced in V3.0RC01 Referable/displayName, it is also poss
...
+
+* Application Class:        	0173-1#01-AAB572#006 Electric engineering, automation, process control engineering
+* Classification Level 1:  	 0173-1#01-AAB583#008 Electrical drive 
+* Classification Level 2:    0173-1#01-AJZ695#008 Servo motor 
+* Classification Level 3:  	 0173-1#01-AJZ696#015 Servo asynchronous motor
+
+For example the semanticId of a submodel containing the properties for the asynchronous servo motor without any further hierarchical modelling would be:
+
+   
+  <aas:semanticId>
```
Comment (BirgitBoss) [inline README.md:182]: formatting not correct, should be <aas:semanticId>

**Review by BirgitBoss — COMMENTED** [review]
2021-05-03: reviewed in Exemplary Modelling WG - no final decision on how to deal with the problems. See comments added to example.

File: README.md [hunk 3]
```diff
@@ -162,8 +162,42 @@ Via display names (introduced in V3.0RC01 Referable/displayName, it is also poss
...
+      
+      <aas:key idType="IRDI" type="GlobalReference">0173-1#01-AJZ696#015</aas:key>
+      
+    </aas:keys> 
+    
+  </aas:semanticId>
+    
+
+
+(Answered: 2021-01-11)
```
Comment (StenGruener) [inline README.md:200]: Add a reference to the screencast #53 of https://admin-shell-io.com/screencasts/

**Comment by StenGruener** [comment]
on hold -> waiting for a joint publication of ECLASS and PI4.0

**Comment by StenGruener** [comment]
image

Q: Should one use application classes in define semantics of a submodel?
A: No, see the cited document (https://eclass.eu/fileadmin/Redaktion/pdf-Dateien/Broschueren/2021-06-29_Whitepaper_PlattformI40-ECLASS.pdf). New defined elements in ECLASS to define Submodels should be used once available.
The only exception is to use application/classification classes in submodels is to denote the category of a product (see ProductClassId property example within the TechnicalData Submodel template).

**Comment by StenGruener** [comment]
question will be re-formulated based on the discussion on 1.8.

**Pull request closed without merging**